lib/pull_request.rb

```diff
@@ -125,14 +129,31 @@ def tell_dependency_manager_what_dependencies_are_allowed
   end
 
   def tell_dependency_manager_what_dependabot_is_changing
+    dependency_updates = commit_message.scan(/Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
+
+    # Commit messages can have different formats
+    if dependency_updates.empty?
+      dependency_updates = commit_message.scan(/Bump (?:\[.*?\]\(.+?\) )?(\w+) from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
```

Comment:
Can this be simplified to this...?

```suggestion
      dependency_updates = commit_message.scan(/Bump (\w+) from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/)
```